lib/Twig/TemplateWrapper.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) 2016 Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Exposes a template to userland.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class Twig_TemplateWrapper
```

Comment:
Changing the meaning of Twig_Template between 1.x and 2.x means it is very hard to write forward-compatible code. So -1 for this.